core/templates/dev/head/pages/base.html

```diff
@@ -72,6 +72,8 @@
       {% include 'pages/header_css_libs.html' %}
     {% endblock header_css %}
 
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/load_constants.js"></script>
```

Comment:
Is this guaranteed to run/load prior to Angular setting oppia.constant('CONSTANTS')?